### DISCUSSION TITLE
Digits sind nicht auf einer Höhe - Stromzähler

### DISCUSSION THREAD

**Discussion by pimp1310** [jomjol/AI-on-the-edge-device#2276] [General]
hi,

da mir so super nett und toll geholfen wurde **(dafür nochmals vielen dank)** bei meinem Wasserzähler wollte ich mal ein zwei fragen los werden bezüglich meines Stromzählers.

ich habe mir diese Vorrichtung mit Spiegel gedruckt, und habe diese dann auch noch 3-4 mal angepasst, ich habe dort nun eine externe Beleuchtung per 2x ws2812B.

![model](https://user-images.githubusercontent.com/6306057/229201600-4752ee8f-81aa-4a2f-b127-cfefe35ce3e6.jpg)

so das ganze funzt auch ganz gut denke ich vom bild her,

![reference_power](https://user-images.githubusercontent.com/6306057/229201871-2e6a5bb7-3632-4db9-9ecb-0676f36514b7.jpg)

aber ich habe das problem das die digits nicht auf einer höhe sind, es sieht so aus würde er das bild etwas verzerren, bzw durchbiegen.

das sieht man hier sehr schön, die zahlen sind nicht auf einer ebene, soll ich jetzt jedes digit in der höhe von hand einstellen? Nur dann warnt die UI ja das das zu schlechten Ergebnissen führen kann.

![problem_powermeter](https://user-images.githubusercontent.com/6306057/229201064-17a07d1f-355e-483b-95d5-619d1d09bc91.jpg)

und in der overview ausserhalb der festlegung des reference Images, sieht das bild wesentlich schlechter aus, hat das einen bestimmten grund?
Oben sieht man das bild während der festlegung des Referenz Bildes.

![problem_powermeter_overview](https://user-images.githubusercontent.com/6306057/229202346-5f742c0a-7ddb-47b2-be4d-ffa9249a5a90.jpg)

Danke

**Comment by ghost** [comment]
Es ist kein Problem, wenn die Ziffern etwas versetzt sind, aber richte die ROIs besser aus, insbesondere wenn du ein dig-cont oder dig-class100 - Model eingestellt hast. Siehe Abschnitt _"Setup using dig-class100 or dig-cont Models"_

https://jomjol.github.io/AI-on-the-edge-device-docs/ROI-Configuration/

<img width="447" alt="image" src="https://user-images.githubusercontent.com/412645/229305319-290d7d19-b86a-48b8-ab99-d502be575098.png">

**Reply by pimp1310** [reply]
hi, die inneren sehen doch gut aus bei mir oder was meinst du? und die äusseren kann ich ja nicht in der größe beeinflussen ohne die inneren, oder irre ich mich?

**Reply by ghost** [reply]
Wenn du "Lock aspect ratio" abwählst kannst du die Höhe der ROIs unabhängig von der Breite beeinflussen.
